CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+# Code of Conduct
+
+## When something happens
+
+If you see a violation of our community's standards of [Conduct](#conduct), follow these steps:
+
+1. Let the person know that what they did is not appropriate and ask them to stop and/or edit their messages.
+2. That person should immediately stop the behavior and correct the issue.
+3. If this doesn’t happen, or if you're uncomfortable speaking up, contact the maintainers.
```

Comment:
@cart 
We should probably add a link to contact you directly for now. What's the best way of getting in touch?